frontend/src/layouts/rjsf-form-layout/RjsfFormLayout.jsx

```diff
@@ -53,6 +53,9 @@ function RjsfFormLayout({
 
   const uiSchema = {
     "ui:classNames": "my-rjsf-form",
+    mark_horizontal_lines: {
+      "ui:widget": !formData?.mark_vertical_lines ? "hidden" : undefined,
+    },
```

Comment:
@jagadeeswaran-zipstack Instead of hardcoding the `mark_horizontal_lines` field, can we automate this? For example, we could check for any fields that are hidden and automatically add them to the `uiSchema`. This way, it becomes a one-time effort, and we won't have to update the `uiSchema` again if we decide to hide additional fields in the future.

Reply:
Yes, It is possible. We can pass uiSchema from backend but right now SDK is not written in a way to support uiSchema being passed down. It will function as it does today on the FE side.